### PR TITLE
Obg 240 fintech fixes

### PR DESCRIPTION
**Pull request by electronicpeter**

### CODE REVIEW

File: fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/AccountService.java [hunk 1]
```diff
@@ -1,52 +1,47 @@
...
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
 
 @Service
 @Slf4j
 @RequiredArgsConstructor
-public class AccountService {
-    @Value("${mock.tppais.listaccounts}")
+public class AccountService extends HandleAcceptedService {
+    @Value("${mock.tppais.listaccounts:#{null}}")
```
Comment (valb3r) [inline fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/AccountService.java]: Can be simplified to
```
@Value("${mock.tppais.listaccounts:false}")
private boolean mockTppAis;
```

File: fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/TransactionService.java [hunk 2]
```diff
@@ -1,56 +1,62 @@
...
 
 import java.time.LocalDate;
 
 @Service
 @Slf4j
 @RequiredArgsConstructor
-public class TransactionService {
-    @Value("${mock.tppais.listtransactions}")
+public class TransactionService extends HandleAcceptedService {
+    @Value("${mock.tppais.listtransactions:#{null}}")
```
Comment (valb3r) [inline fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/TransactionService.java]: Can be simplified to
```
@Value("${mock.tppais.listtransactions:false}")
private boolean mockTppAis;
```

File: fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/AccountService.java (hunk 1 above)
Comment (electronicpeter) [reply fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/AccountService.java]: done

File: fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/TransactionService.java (hunk 2 above)
Comment (electronicpeter) [reply fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/TransactionService.java]: done

**Pull request merged**